README.md

```diff
@@ -1 +1,60 @@
-# kex
\ No newline at end of file
+The purpose of kex is to provide an opinionated executor into a kubernetes container.  Traditionally, if one wanted to exec into a kubernetes container, one had to `kubectl get pods --namespace foo`, visually identify the pod of interest, copy that pod to the buffer, and then `kubectl --namespace foo exec -it <paste_buffer> bash` to exec in to the pod.  This simple utility aims to provide a namespace-specific pod selector for quick execution.
+
+# kex
+
+kex is a pod exec utility.
+
+```
+USAGE:
+  kex             : exec in to the default pod
+  kex ls,list     : list the available pods
+  kex <NUM>       : exec into a specified pod
```

Comment:
@lypht would <NUM> be 1, 2, 3 (for sidecar, etc)? If so, we can get the line number by piping to `sed -n ${NUM}p`

So just to exec (assuming bash), a one-liner would be:
```bash
kubectl exec -it $(kubectl get pods -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}' | sed -n ${NUM}p) bash
```

But we may want to pass a <COMMAND> param too.